crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -436,7 +434,8 @@ where
         // prepass shader
         let fragment_required = !targets.is_empty()
             || ((key.mesh_key.contains(MeshPipelineKey::ALPHA_MASK)
-                || blend_key == MeshPipelineKey::BLEND_PREMULTIPLIED_ALPHA)
+                || blend_key == MeshPipelineKey::BLEND_PREMULTIPLIED_ALPHA
+                || blend_key == MeshPipelineKey::BLEND_ALPHA)
```

Comment:
I think you could also use
```rust
blend_key.intersects(MeshPipelineKey::BLEND_PREMULTIPLIED_ALPHA | MeshPipelineKey::BLEND_ALPHA)
```

Reply:
We cannot do that, see https://github.com/bevyengine/bevy/pull/7291#discussion_r1124953466. But it might be good to add a comment stating why we have to do it like that, as it isn't intuitive.

Reply:
Ah, I see. Yeah, that comment might be necessary. I could foresee someone refactoring this and accidentally breaking something.